doc/registrant-api/v1/contact.md

```diff
@@ -151,33 +155,28 @@ HTTP/1.1 200
 Content-Type: application/json
 
 {
-  "uuid": "84c62f3d-e56f-40fa-9ca4-dc0137778949",
-  "domain_names": ["example.com"],
+  "id": "84c62f3d-e56f-40fa-9ca4-dc0137778949",
```

Comment:
Given the vision (with which I don't agree, but that's beside the point), right now you have the situation where:

- contact update returns UUID as ID and does not return ID
- every other controller returns ID as ID and UUID as UUID.

One of these needs to be changed. I'll leave it to your discretion which one.

Reply:
`id` is a standard name for unique identifiers. Stripe, Facebook, Google and I bet many other service providers use `id` to represent it. If we decided to employ UUIDs, why should we expose auto-incremented primary keys from a database?

I think we should change other controllers in a separate ticket later, so that they return UUID in `id` field, and do not return primary keys at all.

Reply:
> I think we should change other controllers in a separate ticket later, so that they return UUID in id field, and do not return primary keys at all.

Given that this is the last ticket before the API is complete, I'm not certain that is the right approach.